core/src/test/java/org/apache/calcite/test/SqlToRelTestBase.java

```diff
@@ -585,13 +588,16 @@ protected TesterImpl(DiffRepository diffRepos) {
      * @param clusterFactory Called after a cluster has been created
      */
     protected TesterImpl(DiffRepository diffRepos, boolean enableDecorrelate,
-        boolean enableTrim, boolean enableLateDecorrelate,
+        boolean enableTrim,
```

Comment:
is this some kind of auto style format ?

Reply:
Fixed. Keep old format of the parameters.